src/datatypes/Card.ts

```diff
@@ -144,11 +144,16 @@ export type FilterValues = {
 export const boardTypes = ['mainboard', 'maybeboard'] as const;
 export type BoardType = (typeof boardTypes)[number];
 
+export type CubeCardChange = { index: number; oldCard: Card };
```

Comment:
This probably isn't necessary with the end state of removeCardByIndexFromChangeset. Originally I had it returning the modified array because it used filter() instead of findIndex + splice. But typescript didn't like the filter() syntax because the various add/edits/removes/swaps arrays are documented as possibly undefined.